.github/actions/setup-kubernetes/action.yaml

```diff
@@ -77,13 +77,14 @@ runs:
       set -euExo pipefail
       shopt -s inherit_errexit
       
-      export KUBERNETES_PKG_VERSION=${{ inputs.kubernetesVersion }}-00
+      export KUBERNETES_VERSION_SHORT=$( echo ${KUBERNETES_VERSION} | cut -d'.' -f1-2 )
+      export KUBERNETES_PKG_VERSION=${{ inputs.kubernetesVersion }}-1.1
```

Comment:
I get the change in the revision format, but why the change of the revision itself?

Reply:
Looks like numbering starts from 1.1:
```
$ sudo apt-cache policy kubectl
kubectl:
  Installed: 1.25.9-00
  Candidate: 1.27.5-1.1
  Version table:
     1.27.5-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.4-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.3-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.2-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.1-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.0-2.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages